src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -121,18 +135,17 @@ class BaseExpensiTextInput extends Component {
             innerRef,
             autoFocus,
             multiline,
+            secureTextEntry,
             ...inputProps
         } = this.props;
 
         const hasLabel = Boolean(label.length);
         return (
             <View>
-                <View
-                    style={[
-                        !multiline && styles.componentHeightLarge,
-                        ...containerStyles,
-                    ]}
-                >
+                {Boolean(errorText) && (
```

Comment:
Hmm, Did you mistakenly moved the error message to the top of the Input?

Reply:
probably. git made a complete mess of this when merging during rebase

Reply:
it was duplicated rather than moved